cheroot/wsgi.py

```diff
@@ -276,6 +278,11 @@ def get_environ(self):
 
         return env
 
+    def _get_peer_uid(self):
```

Comment:
It isn't obvious what this function purpose is, so it's a good practice to leave comments + docstring.